icicle-cpu/src/exec/interpreter.rs

```diff
@@ -842,7 +842,7 @@ macro_rules! impl_eval_int_single_op {
     };
 }
 
-impl_eval_int_single_op! { IntNegate, (a, (-a.to_signed()).to_unsigned()) }
+impl_eval_int_single_op! { IntNegate, (a, (a.to_signed().wrapping_neg()).to_unsigned()) }
```

Comment:
I thought I'd fixed this, but looks I never merged the change. I believe this is actually equivalent to:

```suggestion
impl_eval_int_single_op! { IntNegate, (a, a.wrapping_neg()) }
```